agent/ansible/roles/pbench-agent-config/tasks/main.yml

```diff
@@ -1,5 +1,17 @@
 ---
 # pbench agent configuration
+- debug:
```

Comment:
Do these `debug` tasks run only when some verbosity level of ansible is requested?

Reply:
Yes, it requires `-v` - else they are skipped.